start-a-local-cluster.md

```diff
@@ -29,11 +29,12 @@ Also, feel free to watch this process in action before going through the steps y
 
 ## Step 1. Start the first node
 
+{% include copy-clipboard.html %}
 ~~~ shell
 $ cockroach start --insecure \
 --host=localhost
 ~~~
-
+{% include copy-clipboard.html %}
```

Comment:
Can you remove the include here, since the block shows a command response that users won't need to copy?